Assignment 5.Rmd

```diff
@@ -66,16 +72,28 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
 #Upload new data
 
-D2 <- 
+D2 <- read.csv("intelligent_tutor_new.csv")
 
 #Generate predicted advice using the predict() command for new students based on tree generated from old students
 
-D2$prediction <- 
+D2$prediction <- predict(score_ctree,D2)
 
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
-
+```{r}
+D2$advice <- ifelse(D2$score > 0.4,
+                    ifelse(D2$score > 0.8,
+                           "no action", "teacher should monitor student progress"),
+                    "teacher should intervene"
+                    )
+#difference between observed and predicted
+difference_obs_pred <- table(D2$advice, D2$prediction)
+difference_obs_pred 
+#Calculate accuracy 
+accuracy <- difference_obs_pred [1,1]/sum(difference_obs_pred [1, ])
+accuracy
+```
```

Comment:
In general, good job! It would be great if you could go further to interpret your model and comment on whether you think the current accuracy rate is an acceptable one and why.